api/wallet.go

```diff
@@ -367,22 +367,53 @@ func (api *API) walletSeedsHandler(w http.ResponseWriter, req *http.Request, _ h
 
 // walletSiacoinsHandler handles API calls to /wallet/siacoins.
 func (api *API) walletSiacoinsHandler(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
-	amount, ok := scanAmount(req.FormValue("amount"))
-	if !ok {
-		WriteError(w, Error{"could not read 'amount' from POST call to /wallet/siacoins"}, http.StatusBadRequest)
-		return
-	}
-	dest, err := scanAddress(req.FormValue("destination"))
-	if err != nil {
-		WriteError(w, Error{"error after call to /wallet/siacoins: " + err.Error()}, http.StatusBadRequest)
-		return
+	var txns []types.Transaction
+	if req.FormValue("destination") != "" {
+		// single amount + destination
+		amount, ok := scanAmount(req.FormValue("amount"))
+		if !ok {
+			WriteError(w, Error{"could not read 'amount' from POST call to /wallet/siacoins"}, http.StatusBadRequest)
+			return
+		}
+		dest, err := scanAddress(req.FormValue("destination"))
+		if err != nil {
+			WriteError(w, Error{"error after call to /wallet/siacoins: " + err.Error()}, http.StatusBadRequest)
+			return
+		}
+		txns, err = api.wallet.SendSiacoins(amount, dest)
+		if err != nil {
+			WriteError(w, Error{"error after call to /wallet/siacoins: " + err.Error()}, http.StatusInternalServerError)
+			return
+		}
+	} else {
+		// multiple amounts + destinations
+		amountStrs := strings.Split(req.FormValue("amounts"), ",")
+		destStrs := strings.Split(req.FormValue("destinations"), ",")
```

Comment:
hmm, I guess we could let SendSiacoinsMulti do the checking, but I'm not 100% sure what is best.